src/NzbDrone.Core/Indexers/Newznab/NewznabCategoryFieldOptionsConverter.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web.UI.WebControls;
+using NzbDrone.Core.Annotations;
+
+namespace NzbDrone.Core.Indexers.Newznab
+{
+    public static class NewznabCategoryFieldOptionsConverter
+    {
+        public static List<FieldSelectOption> GetFieldSelectOptions(List<NewznabCategory> categories)
+        {
+            // Ignore categories not relevant for Sonarr
+            var ignoreCategories = new[] { 0, 1000, 2000, 3000, 4000, 6000, 7000 };
```

Comment:
@Taloth can 7000 be _unignored_? Some indexers only have this, while others are a mess and mark 90% of uploads as 'Other'.

Reply:
Hang on a moment.
The Jackett torznab caps category table has 8000 as Books and 7000 as Other, which seems to contradict the current docs at https://newznab.readthedocs.io/en/latest/misc/api/#predefined-categories
This has been this way since Jackett was created over 5 years ago
It may be we need to amend Jackett to correct its table!

Reply:
Yeah, 7000 is books for pretty much every indexer I know of. There are some differences in the special categories between default newznab and nzedb, but the primary ones are all the same.

Reply:
https://github.com/Jackett/Jackett/pull/9986

Reply:
When I implemented this I was actually considering to return the full list of categories if the filtered list was empty, but I kinda forgot.